CoreArrangements/src/au/gov/asd/tac/constellation/arrangements/planar/NodeType.java

```diff
@@ -21,7 +21,7 @@
  */
 enum NodeType {
 
-    PNode,
-    QNode,
-    LeafNode,
+    PNODE,
+    QNODE,
+    LEAFNODE,
```

Comment:
`LEAF_NODE` ?

Reply:
The changes have been made apologies for the delay.